From 0643804b4e76c0047df1d61ebee1adfe4696d9d4 Mon Sep 17 00:00:00 2001
From: Nils Philippsen <nils@redhat.com>
Date: Wed, 3 Nov 2010 17:54:24 +0100
Subject: [PATCH] patch: SCX4500W

Squashed commit of the following:

commit d3d53225c2a5ee271141bd058e628ffa66d0c065
Author: Nils Philippsen <nils@redhat.com>
Date:   Wed Nov 3 17:53:32 2010 +0100

    add USB id for SCX-4500W
---
 doc/descriptions/xerox_mfp.desc |    4 ++++
 1 files changed, 4 insertions(+), 0 deletions(-)

diff --git a/doc/descriptions/xerox_mfp.desc b/doc/descriptions/xerox_mfp.desc
index 406510f..6b8aa5f 100644
--- a/doc/descriptions/xerox_mfp.desc
+++ b/doc/descriptions/xerox_mfp.desc
@@ -48,6 +48,10 @@
 :usbid "0x04e8" "0x3426"
 :status :good
 
+:model "SCX-4500W"
+:interface "USB"
+:usbid "0x04e8" "0x342b"
+
 :model "CLX-3170fn"
 :interface "USB"
 :usbid "0x04e8" "0x342a"
-- 
1.7.3.2

